src/GuppyDataExplorer/ExplorerSurvivalAnalysis/utils.js

```diff
@@ -22,10 +22,10 @@ export const getXAxisTicks = (data, step = 2, endtime = undefined) => {
  * Filter survival by start/end time
  * @param {SurvivalData[]} data
  * @param {number} startTime
- * @param {number} endTime
+ * @param {number} [endTime]
  * @returns {SurvivalData[]}
  */
-export const filterSurvivalByTime = (data, startTime, endTime) =>
+export const filterSurvivalByTime = (data, startTime, endTime = Infinity) =>
```

Comment:
@stevekrasinsky You got it right. Specifically, here the `endTime` param is used to filter out data points with `time` value greater than `endTime` value. By setting it to `Infinity` when user input is `undefined`, we can make sure that the plots display all survival/risktable data, which is equivalent to using the maximum time from data to set the x-axis domain.